.github/workflows/generate-embeddings.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18]
+        node-version: ['20.19.0']
```

Comment:
We can leave it at `20` which currently pulls in 20.19, and will automatically update to the new minor versions as they are released.